src/caret_analyze/architecture/architecture_loaded.py

```diff
@@ -1062,8 +1095,11 @@ def _is_service_only_callbackgroup(cbg: CallbackGroupValue):
                     cbg_name = cbg.callback_group_name
             else:
                 cbg_name = cbg.callback_group_name  \
-                    or f'{node.node_name}/service_only_callback_group_' \
-                    + '{len(_srv_only_cbg_dict)}'
+                    or f'{node.node_name}/service_only_callback_group_{len(_srv_only_cbg_dict)}'
+                # MYK これはたぶん既存のバグ
```

Comment:
Please avoid using Japanese

Reply:
I corresponded.

Reply:
@miyakoshi-dev 
Please make such comments when you push new code. Otherwise, reviewers cannot confirm new code.
(I think you fixed it in your local PC. If you are kindly sharing your progress, please just add detail, like "I fixed code in my local PC, new code will be uploaded soon")

Reply:
I'm sorry. The code is currently in a half-finished state, so I will commit the new code tomorrow as soon as it is resolved.